PosteAPI/views.py

```diff
@@ -489,18 +490,28 @@ def patch(self, request, id):
         try:
             post = Post.objects.get(pk=id)
             data = json.loads(request.body.decode('utf-8'))
-            post.edit(data.get('title'), data.get('description'), data.get('url'))
+            tagsMerged = data.get('tags')
+            tag_names = [tag.strip() for tag in tagsMerged.split(', ') if tag.strip()]
+            tag_list = []
+            if tag_names:
+                for tag_name in tag_names:
+                    tag, created = Tag.objects.get_or_create(name=tag_name)
+                    print(f"Tag: {tag}, Created: {created}")
+                    tag_list.append(tag)
+            post.edit(data.get('title'), data.get('description'), data.get('url'), tag_list)
             return Response({"success": True}, status=status.HTTP_200_OK)
         except ObjectDoesNotExist:
             message = "Post does not exist"
+            print(message)
             return Response({
                 "success": False,
                 "errors": {
                     "post": [message]
                 }
             }, status=status.HTTP_400_BAD_REQUEST)
         except Exception as e:
-            message = "Server error occurred while deleting post"
+            message = "Server error occurred while editing post"
+            print(message)
```

Comment:
I know I encouraged printing this message, but we should really print the error itself, too. Something like: `print(f"{message}. Error: {e}")` will *also* print the actual error, which is important. Can you include this for both exceptions before merging?

Reply:
sure thing